libs/ui/tabs/brain/src/lib/brn-tabs-trigger.directive.ts

```diff
@@ -8,24 +8,26 @@ import { BrnTabsDirective } from './brn-tabs.directive';
 		'[id]': 'labelId',
 		type: 'button',
 		role: 'tab',
-		'[tabindex]': '_isSelected() ? "0": "-1"',
-		'[attr.aria-selected]': '_isSelected()',
+		'[tabindex]': 'selected() ? "0": "-1"',
+		'[attr.aria-selected]': 'selected()',
 		'[attr.aria-controls]': 'contentId',
-		'[attr.data-state]': "_isSelected() ? 'active' : 'inactive'",
+		'[attr.data-state]': "selected() ? 'active' : 'inactive'",
 		'[attr.data-orientation]': '_orientation()',
 		'[attr.data-disabled]': "disabled ? '' : undefined",
 		'(click)': 'activate()',
```

Comment:
We should probably start adding the `exportAs` for all of the brains 🧠 Thoughts?

Reply:
Yea we definitely can. I think makes sense for some use cases where you just need some simple values to use in the template. Is probably just good to do to make them ever more flexible

Reply:
sweet!

Reply:
can you add a Closes #105 in the commit for this? Thanks already!

Reply:
Done and added export as to rest of brn tab directives. Let me know if anything else